pkg/apis/authentication/v1/defaults.go

```diff
@@ -25,7 +25,8 @@ func addDefaultingFuncs(scheme *runtime.Scheme) error {
 	return RegisterDefaults(scheme)
 }
 
-func SetDefaults_TokenRequestSpec(obj *authenticationv1.TokenRequestSpec) {
+//SetDefaultsTokenRequestSpec sets a default expiration for a given TokenRequestSpec
+func SetDefaultsTokenRequestSpec(obj *authenticationv1.TokenRequestSpec) {
```

Comment:
this will not work. the generator for the defaults needs: `SetDefaults_TokenRequestSpec`
we cannot disable the linter for this package.

the `zz_` files are created using `./hack/update-codegen.sh`

/close